pkgs/top-level/all-packages.nix

```
@@ -33277,7 +33277,9 @@ with pkgs;
 
   pgmanage = callPackage ../applications/misc/pgmanage { };
 
-  pgadmin = callPackage ../applications/misc/pgadmin {
+  pgadmin4 = callPackage ../tools/admin/pgadmin { };
+
+  pgadmin3 = callPackage ../applications/misc/pgadmin {
```

Both packages should be in the same directory and only differ in the name of the nix file. I suggest we keep 4 as default.nix and name 3 as 3.nix.

```suggestion
  pgadmin_4 = callPackage ../tools/admin/pgadmin { };

  pgadmin_3 = callPackage ../applications/misc/pgadmin {
```